### PR TITLE
Is not a Mass

### PR DESCRIPTION
**Pull request by julioalucero**
@eloyesp @olvap @antico5 @noili @hsulionel 

Si al menos alguno le podría pegar una mirada se lo agradezo, gracias totales

### CODE REVIEW

File: _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Is not a Mass, is just a stand-up"
```
Comment (eloyesp) [inline _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown]: "Mass" is not clear enough, may be use the word *rite* or *sacrament*.

File: _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+date:   2016-12-30 11:28:00 -0300
+categories: software, stand-up, scrum
+---
+
+Each day, at 12.00 am, the priest is ready to start the mass. It doesn’t really matter what you did or not the last day or even the quality of the code, the only most important thing is that you be there on the stand-up and talk about something.
```
Comment (eloyesp) [inline _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown]: The word priest is very catholic, may be pastor, lord or minister could work better.

File: _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Bosses, PMs, whatever has to re-organize his priority, there are at least 10 different most important things in a project than this meeting. 
+
+> “Please inform with at least 34.3 hours before if you can’t assist a
```
Comment (eloyesp) [inline _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown]: 36 sounds better.

File: _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+> stand-up”
+
+Is not a Sin and doesn’t matter the reason. 
+
+Not to be strict in this kind of thinks helps to generate a good atmosphere in the project, which in consequence helps to keep you motivated and this last adjective believe me that is something that the client is interested in.
```
Comment (eloyesp) [inline _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown]: Could you refrase "Not to be strict" with something positive, like "being flexible"?

**Review by eloyesp — CHANGES_REQUESTED** [review]
I like it, but I think it could be improved.

File: _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown (hunk 1 above)
Comment (julioalucero) [reply _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown]: Let's go with Sacrament

File: _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown (hunk 2 above)
Comment (julioalucero) [reply _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown]: Minister is ok to me

File: _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown (hunk 4 above)
Comment (julioalucero) [reply _posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown]: cool, yes. I have to learn to not say non negative things. I said it even in interviews

File: _posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+Each day, at 12.00 am, the Minister is ready to start the Sacrament. It doesn’t really matter what you did or not the last day or even the quality of the code, the only most important thing is that you be there on the stand-up and talk about something.
+
+Bosses, PMs, whatever has to re-organize his priority, there are at least 36 different most important things in a project than this meeting. 
+
+> “Please inform with at least 34.3 hours before if you can’t assist a
+> stand-up”
+
+Is not a Sin and doesn’t matter the reason. 
+
+Being flexible about meetings helps to generate a good atmosphere in the project, which in consequence helps to keep you motivated and this last adjective believe me that is something that the client is interested in.
```
Comment (olvap) [inline _posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown]: que te parece cambiar este ultimo parrafo por:

Being flexible about meetings helps to generate a good atmosphere in the project, which in consequence help you to keep motivated and this, believe me, is something that the client is interested in.

File: _posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Is not a Sacrament, is just a stand-up"
+date:   2016-12-30 11:28:00 -0300
+categories: software, stand-up, scrum, en
+---
+
+Each day, at 12.00 am, the Minister is ready to start the Sacrament. It doesn’t really matter what you did or not the last day or even the quality of the code, the only most important thing is that you be there on the stand-up and talk about something.
+
+Bosses, PMs, whatever has to re-organize his priority, there are at least 36 different most important things in a project than this meeting.
```
Comment (antico5) [inline _posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown]: Algunas correcciones:
..the only most important thing is that you **are** there..
Bosses, PMs, **whatever**, **have** to re-organize..
..there are at least 36 different **more** important things..

Muy bueno

Comment (julioalucero) [reply _posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown]: thanks

File: _posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown (hunk 5 above)
Comment (julioalucero) [reply _posts/2016-12-30-Is-not-a-Sacrament-is-just-a-stand-up.markdown]: thanks

**Pull request merged**